index.js

```diff
@@ -10,6 +10,8 @@ var streams = require('./lib/stream')
 const SCHEMA_READING_STREAM = 'schema_reading_stream'
 const SCHEMA_POSTING_STREAM = 'schema_posting_stream'
 
+const isABib = ((process.env.RETRIEVAL_TYPE === 'bib'))
```

Comment:
@EdwinGuzman,

I think it was there in the old commit, the new one doesn't have that. I will squash my commits next time.